lib/bgsite_official_web/live/websites_like.ex

```diff
@@ -14,21 +15,24 @@ defmodule BgsiteOfficialWeb.WebsitesLike do
   @impl true
   def mount(params, %{"admin_token" => admin_token} = session, socket) do
     tag = Categories.get_tag!(params["id"])
-    websites = tag.websites |> Repo.preload(:tags)
+    website = tag.websites |> Repo.preload(:tags)
     admin = Accounts.get_admin_by_session_token(admin_token)
     socket = assign(
         socket,
         tag: tag,
-        websites_for_tag: websites,
+        websites_for_tag: website,
         current_admin: admin
       )
     {:ok, socket}
   end
-  # 
-  # def handle_event("add_like", %{"website-id" => website_id}, socket) do
-  #   Home.bump_site_likes(website_id)
-  #   {:noreply, socket}
-  # end
+
+  def handle_event("add_like", %{"website-id" => website_id}, socket) do
+     Home.bump_site_likes(website_id)
+     current_likes_count = website.likes
```

Comment:
there aint connection between socket assigned website and this `website.likes`, Try this
```
website = socket.assigns[:website]
```
before `current_likes_count = website.likes` :)

Reply:
IMO you are missing to fetch the `website` based on `website_id`. Is there `website` struct on `socket`, actually? 
(fetching `website` by `id`: ```website = Home.get_websites!(website_id)```) Anoying plural again...

Reply:
The idea was:
- to fetch the `current_website_likes` inside this bump function and use it to increase the likes with `1` here: https://github.com/N-Patarov/bgsite_official/blob/add-likes/lib/bgsite_official/home.ex#L27

Now you are requiring `likes` as an argument for the function, we just need `website` to bump its current likes count.

Reply:
I would fix `bump..` like this:

```elixir
def bump_site_likes(%Websites{} = website) do
  updated_likes_count = website.likes + 1
  Home.update_websites(website, %{likes: updated_likes_count})
end
```